src/Discord.Net.Webhook/WebhookClientHelper.cs

```diff
@@ -36,6 +36,21 @@ public static async Task<RestInternalWebhook> GetWebhookAsync(DiscordWebhookClie
             var model = await client.ApiClient.CreateWebhookMessageAsync(client.Webhook.Id, args, options: options).ConfigureAwait(false);
             return model.Id;
         }
+        public static async Task ModifyMessageAsync(DiscordWebhookClient client, ulong messageId, string text, IEnumerable<Embed> embeds,
```

Comment:
Also needs to follow the same pattern as modifying user message like stated above.